src/guiapi/include/MItem_menus.hpp

```diff
@@ -215,3 +215,33 @@ class MI_FOOTER_SETTINGS : public WI_LABEL_t {
 protected:
     virtual void click(IWindowMenu &window_menu) override;
 };
+
+class MI_LANGUAGUE_USB : public WI_LABEL_t {
+    static constexpr const char *const label = N_("Load lang from USB");
```

Comment:
You tagged the string correctly with N_() if it was a production code. 
For debugging code please remove the N_() as gettext doesn't understand #ifdefs and this text could accidentally get into production translation template file.